app/Http/Controllers/Dashboard/IncidentController.php

```diff
@@ -111,14 +111,14 @@ public function createIncidentAction()
     {
         try {
             $incident = $this->dispatch(new ReportIncidentCommand(
-                Binput::get('name'),
-                Binput::get('status'),
-                Binput::get('message'),
-                Binput::get('visible', true),
-                Binput::get('component_id'),
+                Binput::get('incident.name'),
+                Binput::get('incident.status', 0),
```

Comment:
Why are we defaulting the status? That's a value that should *always* be passed through?